src/app/Providers/ProvidersPage.tsx

```diff
@@ -57,8 +68,11 @@ const ProvidersPage: React.FunctionComponent = () => {
   ).filter((type) => !!type) as ProviderType[];
 
   const [activeProviderType, setActiveProviderType] = React.useState<ProviderType | null>(
```

Comment:
Instead of `useState` here, we should probably just use `const activeProviderType = `match?.params.providerType || null` and then everywhere we have `setActiveProviderType` we should instead redirect to the route for that provider type. That way the route is the single source of truth. Right now if you change tabs, go somewhere, and click Back it won't remember the tab change.

Reply:
That's a good point.